### PR TITLE
Fix errcheck linter for all non-test files

### PR DESCRIPTION
**Pull request by panslava**
Add handling errors where it was missing

### CODE REVIEW

File: cmd/echo/app/handlers.go [hunk 1]
```diff
@@ -82,15 +85,23 @@ func RunHTTPServer(ctx context.Context) {
...
 
 	<-ctx.Done()
 }
 
 func healthCheck(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
-	w.Write([]byte("health: OK"))
+	_, err := w.Write([]byte("health: OK"))
+	if err != nil {
+		klog.Errorf("Error writing bytes %v", err)
```
Comment (kl52752) [inline cmd/echo/app/handlers.go:102]: can you log r.UserAgent(), r.RemoteAddr so we know what this error refers to?

File: cmd/echo/app/handlers.go [hunk 2]
```diff
@@ -123,7 +134,11 @@ func echo(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	w.Write(processedData)
+	_, err = w.Write(processedData)
+	if err != nil {
+		klog.Errorf("Error writing data: %v", err)
```
Comment (kl52752) [inline cmd/echo/app/handlers.go:139]: can you log r.UserAgent(), r.RemoteAddr so we know what this error refers to?

File: cmd/fuzzer/fuzzer.go [hunk 3]
```diff
@@ -43,14 +44,17 @@ func main() {
 
 	switch os.Args[1] {
 	case "validate":
-		app.ValidateFlagSet.Parse(os.Args[2:])
+		err := app.ValidateFlagSet.Parse(os.Args[2:])
+		if err != nil {
+			klog.Errorf("Error parsing os.Args: %v", err)
```
Comment (kl52752) [inline cmd/fuzzer/fuzzer.go:49]: maybe you can log args[2:] or at least the size of this slice?

File: cmd/fuzzer/fuzzer.go [hunk 4]
```diff
@@ -43,14 +44,17 @@ func main() {
...
+		}
 	default:
 		flag.Usage()
 		os.Exit(1)
 	}
 
 	// Suppress klog logging before flag.Parse() error.
-	flag.CommandLine.Parse([]string{})
+	_ = flag.CommandLine.Parse([]string{})
```
Comment (kl52752) [inline cmd/fuzzer/fuzzer.go:57]: this hack does not make sense. Maybe you should disable Lint in this place?

File: cmd/glbc/app/handlers.go [hunk 5]
```diff
@@ -129,8 +135,9 @@ func putFlag(w http.ResponseWriter, r *http.Request) {
 }
 
 func setVerbosity(v string) {
-	flag.Lookup("v").Value.Set(v)
 	klog.V(0).Infof("Setting verbosity level to %q", v)
+	err := flag.Lookup("v").Value.Set(v)
+	klog.Errorf("Error setting verbosity level: %v", err)
```
Comment (kl52752) [inline cmd/glbc/app/handlers.go:140]: you forget about if `err != nil`

File: pkg/experimental/workload/controller.go [hunk 6]
```diff
@@ -132,7 +132,7 @@ func NewController(ctx *ControllerContext) *Controller {
 }
 
 func (c *Controller) Run(stopCh <-chan struct{}) {
-	wait.PollUntil(5*time.Second, func() (bool, error) {
+	_ = wait.PollUntil(5*time.Second, func() (bool, error) {
```
Comment (kl52752) [inline pkg/experimental/workload/controller.go:135]: as above maybe you should disable lint in this place

File: cmd/echo/app/handlers.go (hunk 2 above)
Comment (panslava) [reply cmd/echo/app/handlers.go]: Thanks, added

File: cmd/fuzzer/fuzzer.go (hunk 3 above)
Comment (panslava) [reply cmd/fuzzer/fuzzer.go]: Thanks, added

File: cmd/glbc/app/handlers.go (hunk 5 above)
Comment (panslava) [reply cmd/glbc/app/handlers.go]: Good catch, thanks, fixed

File: cmd/fuzzer/fuzzer.go (hunk 4 above)
Comment (panslava) [reply cmd/fuzzer/fuzzer.go]: Disabled lint for .Parse

File: pkg/experimental/workload/controller.go (hunk 6 above)
Comment (panslava) [reply pkg/experimental/workload/controller.go]: Disabled lint for .PollUntil

**Pull request merged**